blog/_posts/2022-09-01-scala-3.2.0-released.md

```diff
@@ -0,0 +1,242 @@
+
+The `-Vprofile` option family is useful for trouble-spotting long compile times.
+
+### New experimental APIs
+
+The following new APIs were introduced to the language and currently can be used in experimental code. We are looking forward to user feedback before we decide to stabilise them.
+
+- [`scala.annotation.MainAnnotation`](https://scala-lang.org/api/3.x/scala/annotation/MainAnnotation.html#) - the annotation that can be used to define custom entry points to the application
+- [`scala.annotation.newMain`](https://scala-lang.org/api/3.x/scala/annotation/newMain.html#) - new implementation of `@main` annotation. It is planned to replace it in the future
+- [`scala.util.TupledFunction`](https://scala-lang.org/api/3.x/scala/util/TupledFunction.html#) - the typeclass allowing to abstract over function arities
+
+## Stabilised APIs
+
+The 3.2 release stabilises a bunch of APIs that previously could be used only in experimental code. Those are:
+
+- [`scala.deriving.Mirror.fromTuple`](https://scala-lang.org/api/3.x/scala/deriving/Mirror$.html#fromTuple-fffffb85)
+- [`scala.Tuple.:*`](https://scala-lang.org/api/3.x/scala/Tuple.html#:*-a82) together with [`scala.Tuple.Append`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Append-0)
+- [`scala.NonEmptyTuple.init`](https://scala-lang.org/api/3.x/scala/NonEmptyTuple.html#init-ffffff96) together with [`scala.Tuple.Init`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Init-0)
+- [`scala.NonEmptyTuple.last`](https://scala-lang.org/api/3.x/scala/NonEmptyTuple.html#last-fffff7c4) together with [`scala.Tuple.Last`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Last-0)
+- [`scala.quoted.Quotes.reflectModule.AppliedTypeModule.apply`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$AppliedTypeModule.html#apply-fffffbf2)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.asQuotes`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#asQuotes-fffffdc2)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.termRef`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#termRef-d26)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.typeRef`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#typeRef-d26)
+- [`scala.quoted.Quotes.reflectModule.TypeReprMethods.substituteTypes`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeReprMethods.html#substituteTypes-3a6)
+- [`scala.quoted.Quotes.reflectModule.TypeReprMethods.typeArgs`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeReprMethods.html#typeArgs-5f)
+- [`scala.quoted.Quotes.reflectModule.TypeTreeModule.ref`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeTreeModule.html#ref-d26)
+- [`scala.Predef.eq`](https://scala-lang.org/api/3.x/scala/Predef$.html#eq-fffff7fa)
+- [`scala.Predef.ne`](https://scala-lang.org/api/3.x/scala/Predef$.html#ne-fffff7fa)
```

Comment:
![image](https://user-images.githubusercontent.com/11855167/187649663-36bfda7a-7ad9-4280-88a0-bd5272e261fb.png)
Maybe we can use short names instead of fully qualified names to make it look less overwhelming.

`scala.quoted.Quotes.reflectModule.SymbolMethods.termRef` => `SymbolMethods.termRef`